mllib/src/test/scala/org/apache/spark/ml/classification/ProbabilisticClassifierSuite.scala

```diff
@@ -56,6 +56,21 @@ class ProbabilisticClassifierSuite extends SparkFunSuite {
     val testModel = new TestProbabilisticClassificationModel("myuid", 2, 2)
     assert(testModel.friendlyPredict(Vectors.dense(Array(1.0, 2.0))) === 1.0)
   }
+
+  test("test tiebreak") {
+    val testModel = new TestProbabilisticClassificationModel("myuid", 2, 2)
+      .setThresholds(Array(0.4, 0.4))
+    assert(testModel.friendlyPredict(Vectors.dense(Array(0.6, 0.6))) === 0.0)
+  }
+
+  test("bad thresholds") {
+    intercept[IllegalArgumentException] {
```

Comment:
Actually I think sum to 1 is fine - the R package actually limits to !sum > 1 => sum <= 1 (https://github.com/cran/randomForest/blob/master/R/predict.randomForest.R#L47) - I think I misread it earlier.